tests/common.rs

```diff
@@ -3,10 +3,13 @@
 extern crate fantoccini;
 extern crate futures_util;
 
-use fantoccini::{error, Client};
+use fantoccini::error;
 
```

Comment:
Let's just remove this line here and merge the `use` blocks for `fantoccini` into:
```rust
use fantoccini::{error, Client, ClientBuilder};
```